### PR TITLE
assertutil, etcdutil, grpcutil, keyutil, logutil, metricutil: testify the tests

### PR DESCRIPTION
**Pull request by JmPotato**
Signed-off-by: JmPotato <ghzpotato@gmail.com>

### What problem does this PR solve?

Issue Number: ref #4813.

### What is changed and how does it work?

```commit-message
Testify the pkg/assertutil, pkg/etcdutil, pkg/grpcutil, pkg/keyutil, pkg/logutil, pkg/metricutil tests.
```

### Check List

Tests

- Unit test
- Integration test

### Release note

```release-note
None.
```

### CODE REVIEW

File: pkg/etcdutil/etcdutil_test.go [hunk 1]
```diff
@@ -21,43 +21,35 @@ import (
...
-func (s *testEtcdutilSuite) TestMemberHelpers(c *C) {
+func TestMemberHelpers(t *testing.T) {
 	cfg1 := NewTestSingleConfig()
 	etcd1, err := embed.StartEtcd(cfg1)
 	defer func() {
 		etcd1.Close()
 		CleanConfig(cfg1)
 	}()
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
```
Comment (bufferflies) [inline pkg/etcdutil/etcdutil_test.go:37]: How about use `re:=require.New(t);re.NoError` if there are many assert in one unit test?

Comment (JmPotato) [reply pkg/etcdutil/etcdutil_test.go:37]: Good idea!

Comment (JmPotato) [reply pkg/etcdutil/etcdutil_test.go:37]: What about rewriting this way in another PR? Let's merge this one first.

**Review by bufferflies — APPROVED** [review]

**Review by nolouch — APPROVED** [review]

**Pull request merged**